diff -urN kiax-0.8.5-src.org/qkiax.pro kiax-0.8.5-src/qkiax.pro
--- kiax-0.8.5-src.org/qkiax.pro	2005-04-08 14:29:52.000000000 +0000
+++ kiax-0.8.5-src/qkiax.pro	2006-04-24 18:42:09.268407250 +0000
@@ -3,15 +3,14 @@
 # Subdir relative project main directory: .
 # Target is a subdirs project 
 
-TARGETDEPS += lib/libiaxclient.a \
-              src/../bin/kiax 
+TARGETDEPS += src/../bin/kiax 
 CONFIG += release \
           warn_on \
           qt \
           thread \
           x11 
 TEMPLATE = subdirs 
-SUBDIRS += lib \
-           i18n \
+SUBDIRS += i18n \
            src  \
 	   bin
+LIBS += -liaxclient
diff -urN kiax-0.8.5-src.org/src/src.pro kiax-0.8.5-src/src/src.pro
--- kiax-0.8.5-src.org/src/src.pro	2006-03-25 23:21:25.000000000 +0000
+++ kiax-0.8.5-src/src/src.pro	2006-04-24 18:42:09.268407250 +0000
@@ -7,11 +7,11 @@
             icons \
             translations 
 translations.files += ../i18n/*.qm 
-translations.path = /usr/local/share/kiax/i18n/ 
+translations.path = /usr/share/kiax/i18n/ 
 icons.files += ../icons/*.png 
-icons.path = /usr/local/share/kiax/icons/ 
+icons.path = /usr/share/kiax/icons/ 
 binary.files += ../bin/kiax 
-binary.path = /usr/local/bin/ 
+binary.path = /usr/bin/ 
 FORMS += aboutbox.ui \
          logwindow.ui \
          mainwindow.ui \
@@ -86,12 +86,10 @@
            kiaxcontactlistitem.cpp \
            kiaxiconfactory.cpp \
            kiaxkeyfilter.cpp 
-TARGETDEPS += ../lib/libiaxclient.a
-LIBS += ../lib/libiaxclient.a ${LIBOSSAUDIO}
-INCLUDEPATH = ../src \
-../lib
-DEFINES = ICONS_PATH=\"/usr/local/share/kiax/icons/\" \
-I18N_DIRECTORY=\"/usr/local/share/kiax/i18n/\"
+LIBS += -liaxclient ${LIBOSSAUDIO}
+INCLUDEPATH = ../src /usr/include/iaxclient
+DEFINES = ICONS_PATH=\"/usr/share/kiax/icons/\" \
+I18N_DIRECTORY=\"/usr/share/kiax/i18n/\"
 TARGET = ../bin/kiax
 CONFIG += release \
 warn_on \
diff -urN kiax-0.8.5-src.org/src/src.pro.or kiax-0.8.5-src/src/src.pro.or
--- kiax-0.8.5-src.org/src/src.pro.or	2006-03-25 23:21:46.000000000 +0000
+++ kiax-0.8.5-src/src/src.pro.or	2006-04-24 18:42:59.507547000 +0000
@@ -7,11 +7,11 @@
             icons \
             translations 
 translations.files += ../i18n/*.qm 
-translations.path = /usr/local/share/kiax/i18n/ 
+translations.path = /usr/share/kiax/i18n/ 
 icons.files += ../icons/*.png 
-icons.path = /usr/local/share/kiax/icons/ 
+icons.path = /usr/share/kiax/icons/ 
 binary.files += ../bin/kiax 
-binary.path = /usr/local/bin/ 
+binary.path = /usr/bin/ 
 FORMS += aboutbox.ui \
          logwindow.ui \
          mainwindow.ui \
@@ -86,12 +86,10 @@
            kiaxcontactlistitem.cpp \
            kiaxiconfactory.cpp \
            kiaxkeyfilter.cpp 
-TARGETDEPS += ../lib/libiaxclient.a
-LIBS += ../lib/libiaxclient.a ${LIBOSSAUDIO}
-INCLUDEPATH = ../src \
-../lib
-DEFINES = ICONS_PATH=\"/usr/local/share/kiax/icons/\" \
-I18N_DIRECTORY=\"/usr/local/share/kiax/i18n/\"
+LIBS += -liaxclient ${LIBOSSAUDIO}
+INCLUDEPATH = ../src /usr/include/iaxclient
+DEFINES = ICONS_PATH=\"/usr/share/kiax/icons/\" \
+I18N_DIRECTORY=\"/usr/share/kiax/i18n/\"
 TARGET = ../bin/kiax
 CONFIG += release \
 warn_on \
